terraform/nomad-whisper/create-nomad-whisper-configuration.sh

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bash
+set -x 
+unset SSH_USER
+
+[ -z "$POOL_TYPE" ] && export POOL_TYPE="whisper"
+[ -z "$ROLE" ] && export ROLE="whisper-pool"
+[ -z "$MEMORY_IN_GBS" ] && export MEMORY_IN_GBS="240"
+[ -z "$SHAPE" ] && export SHAPE="VM.GPU.A10.1"
+[ -z "$OCPUS" ] && export OCPUS="30"
+[ -z "$BASE_IMAGE_TYPE" ] && export BASE_IMAGE_TYPE="GPU"
+[ -z "$POSTRUNNER_PATH" ] && export POSTRUNNER_PATH="terraform/nomad-whisper/user-data/postinstall-runner-nomad-whisper-oracle.sh"
+
+# IF THE CURRENT DIRECTORY HAS stack-env.sh THEN INCLUDE IT
+[ -e ./stack-env.sh ] && . ./stack-env.sh
+
+if [ -z "$ENVIRONMENT" ]; then
+   echo "No ENVIRONMENT provided or found.  Exiting ..."
+   exit 201
+fi
+
+[ -e ./sites/$ENVIRONMENT/stack-env.sh ] && . ./sites/$ENVIRONMENT/stack-env.sh
+
+LOCAL_PATH=$(dirname "${BASH_SOURCE[0]}")
+
+[ -z "$ROLE" ] && ROLE="nomad-pool"
+[ -z "$POOL_TYPE" ] && POOL_TYPE="general"
```

Comment:
this pool type should be whisper, yes?  or really since we defined it above we can just remove these additions.
```suggestion
```